docs/operator/k8s_guide.md

```diff
@@ -0,0 +1,725 @@
+CONTROLLER_NAME="sealed-secrets-controller" # This can be checked in k8s/Services
+
+# rucio namespace
+RUCIO_NS="rucio"
+
+# Output dir
+SECRETS_STORE="/root/flux-compass/SECRETS/"
+
+
+# name of output secret to apply
+OUTPUT_SECRET="ruciocompass-db.yaml"
+
+kubectl create secret generic ${HELM_RELEASE}-server-hostcert --dry-run=client --from-file=${RAW_SECRETS_SERVERS}/hostcert.pem -o yaml | \
+kubeseal --controller-name=${CONTROLLER_NAME} --controller-namespace=${CONTROLLER_NS} --format yaml --namespace=${RUCIO_NS} > ${SECRETS_STORE}/ss_${OUTPUT_SECRET}
+
+kubectl apply -f ${SECRETS_STORE}/ss_${OUTPUT_SECRET}
+```
+
+Where:
+- `HELM_RELEASE` is the name of the helm release that is being used.
+- `RAW_SECRETS_SERVERS` is the path in which the files that will be used as secrets are stored
+- `CONTROLLER_NS` is the namespace assigned to the secrets manager.
+- `CONTROLLER_NAME` is the name of to the secrets manager controller.
+- `CONTROLLER_NAME` is the name of to the secrets manager controller.
+- `RUCIO_NS` is the namespace related to Rucio (in the COMPASS case, it is called `rucio`).
+- `SECRETS_STORE` path to which the files containing the encrypted secrets will be stored. These files will be used by flux.
+
+`kubectl create secret generic` Creates a secret from the specific file that needs to be transposed to a secret. 
+
+Then `kubeseal` perform the encryption and the encoding, saving the secret to a file stored in `SECRETS_STORE`.
+Finally, `kubectl apply` applies the secret to the cluster. Please notice that is eventually also achieved via flux, by pushing the secret file to the repository.
+
+:::tip[Secret tip]
+IF `kubeseal` returns an error, it's probably because it's not installed:
+```sh
+wget https://github.com/bitnami-labs/sealed-secrets/releases/download/v0.18.0/kubeseal-0.18.0-linux-amd64.tar.gz
+tar xfz kubeseal-0.18.0-linux-amd64.tar.gz
+sudo install -m 755 kubeseal /usr/local/bin/kubeseal
+```
+Adjust the version and the build details according to your machine.
+:::
+The content of the secret will be of this type: 
+```yaml ss_rucio-servers-hostcert.yaml
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: rucio-servers-hostcert
+  namespace: rucio
+spec:
+  encryptedData:
+    hostcert.pem: <encrypted secret>
+  template:
+    data: null
+    metadata:
+      creationTimestamp: null
+      name: rucio-servers-hostcert
+      namespace: rucio
+```
+## Rucio Servers
+[Reference Helm chart](https://gitlab.cern.ch/rucio-it/flux-compass/-/blob/master/sync/rucio-servers.yaml?ref_type=heads).
+
+>The Rucio servers are the backbone of the Rucio system. They handle all core functionalities including data management, rule-based data replication, data placement, and monitoring. The servers ensure the integrity and availability of data across various storage systems.
+
+In order to setup this service, four steps are needed:
+1. Produce the Helm chart
+2. Setup the LB
```

Comment:
Do you define LB before this point? I missed it if so

Reply:
Added acronym in the bullet list: 
> Setup the LoadBalancers (LBs).